samples/cext-trey-research-python/appManifest/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
+  "id": "${{TEAMS_APP_ID}}",
+  "packageName": "com.microsoft.teams.extension",
```

Comment:
Let's remove `packageName` as its not required in 1.19 manifest schema

Reply:
Fixed